bin/fabric.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ ! -z "$CI" ] && [ ! -z "$FABRIC_API_KEY" ]; then
+	echo "Configuring Fabric..."
+	../Frameworks/Fabric/Fabric.framework/run "$FABRIC_API_KEY"
```

Comment:
Xcode's environment variables should also be available here from the run phase, so perhaps this could be:

`"${PROJECT_DIR}"/Frameworks/Fabric/Fabric.framework/run "$FABRIC_API_KEY"`